app/views/case_studies/index.html.haml

```diff
@@ -18,5 +24,15 @@
     .col-md-2
       / WARNING: SIDEBAR UNDER CONSTRUCTION!
     .col-md-10.left-pad
```

Comment:
I would wrap this section in the above `-if current_user.admin? || current_user.editor?` `if` block, so that the html for hidden methods doesn't even get rendered for basic users. Even though basic users aren't able to see the button which un-hides this section, from a strict security standpoint, everything delivered from the server to the browser is user-accessible since any user can open the browser console and inspect/modify html/css/js all by hand